project/dexter-core/src/java/com/samsung/sec/dexter/core/util/FileService.java

```diff
@@ -0,0 +1,9 @@
+package com.samsung.sec.dexter.core.util;
+
+import java.io.File;
+
+public class FileService {
```

Comment:
I think we can use java.nio.file instead of java.io.File.
: Files.exist or not Exists method

It is a good idea to separate methods related Files into FileService class.
However, DexterUtil class has many File-related method, 
so we'd better move them into FileService class soon or later.